src/dash-table/utils/TableClipboardHelper.ts

```diff
@@ -42,6 +42,12 @@ export default class TableClipboardHelper {
         Clipboard.set(e, value);
     }
 
+    public static clearClipboard() {
+        Clipboard.clear();
+        TableClipboardHelper.lastLocalCopy = [];
+        TableClipboardHelper.localCopyWithoutHeaders = [];
+    }
```

Comment:
Deal with the special table copy/paste code -- reset the previsouly set copies -- will pick up the last copy event normally when doing a paste from filter columns.